### PR TITLE
Add 2024 FastSim workflow

### PR DESCRIPTION
**Pull request by srimanob**
#### PR description:
Add 2024 FastSim workflow. This PR used PU dataset from 2023FSPU first, and will update when 2024FSPU is available.

#### PR validation:
Test with workflow 16434.0

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:
To be backport to the release of 2024 MC

### CODE REVIEW

**Comment by srimanob** [comment]
Hi @AdrianoDee 

In https://github.com/cms-sw/cmssw/pull/44075, you add 2024 FastSim workflows (14434.0, 14634.0, 14440.303), but those ID were not for FastSim. Is it just a bug in that PR?

**Comment by srimanob** [comment]
test parameters:
- workflows = 16434.0, 16634.0, 16440.303
- relvals_opt = --what cleanedupgrade,standard,highstats,pileup,generator,extendedgen,production,identity,ged,machine,premix,nano,gpu,2017,2026

**Comment by srimanob** [comment]
@cmsbuild please test

**Comment by srimanob** [comment]
+Upgrade

All new 2024 FastSim workflow run fine.

FYI @sbein @kpedro88

**Comment by srimanob** [comment]
@cmsbuild please test

**Comment by srimanob** [comment]
Kindly ping @cms-sw/pdmv-l2

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -4377,6 +4377,7 @@ def gen2024HiMix(fragment,howMuch):
 defaultDataSets['2024']='CMSSW_14_0_0_pre3-140X_mcRun3_2024_realistic_v1_STD_2024_noPU-v'
 defaultDataSets['2024HLTOnDigi']='CMSSW_14_0_0_pre3-140X_mcRun3_2024_realistic_v1_STD_2024_noPU-v'
 defaultDataSets["2024SimOnGen"] = 'CMSSW_14_0_0_pre3-140X_mcRun3_2024_realistic_v1_STD_2024_noPU-v'
+defaultDataSets['2024FS']='CMSSW_13_0_11-130X_mcRun3_2023_realistic_withEarly2023BS_v1_FastSim-v' #To replace with new dataset
```
Comment (AdrianoDee) [inline Configuration/PyReleaseValidation/python/relval_steps.py:4380]: Hi @srimanob if it is useful to have a new production let us know on JIRA and we can organize this :-)

**Comment by AdrianoDee** [comment]
+pdmv
(failures un-related)

**Comment by srimanob** [comment]
Failure does not relate to this PR.

**Comment by mandrenguyen** [comment]
+1

File: Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py [hunk 2]
```diff
@@ -3102,6 +3104,14 @@ def condition(self, fragment, stepList, key, hasHarvest):
         'BeamSpot': 'DBrealistic',
         'ScenToRun' : ['Gen','Sim','Digi','RecoNano','HARVESTNano','ALCA'],
     },
+    '2024FS' : {
+        'Geom' : 'DB:Extended',
+        'GT' : 'auto:phase1_2024_realistic',
+        'HLTmenu': '@relval2024',
+        'Era' : 'Run3_FastSim',
```
[the remark this answers is not in the document]
Comment (srimanob) [reply Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py:3111]: Yes, it should. Not remember why I made it in that way.

[the remark this answers is not in the document]
Comment (kpedro88) [reply Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py:3111]: Okay, I can submit a PR to update this.

[the remark this answers is not in the document]
Comment (kpedro88) [reply Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py:3111]: #48153

**Pull request merged**